packages/node-experimental/src/sdk/otelContextManager.ts

```diff
@@ -0,0 +1,32 @@
+import type { Context } from '@opentelemetry/api';
+import * as api from '@opentelemetry/api';
+import { AsyncLocalStorageContextManager } from '@opentelemetry/context-async-hooks';
+import type { Carrier, Hub } from '@sentry/core';
+import { ensureHubOnCarrier, getCurrentHub, getHubFromCarrier } from '@sentry/core';
+
+export const OTEL_CONTEXT_HUB_KEY = api.createContextKey('sentry_hub');
+
+function createNewHub(parent: Hub | undefined): Hub {
+  const carrier: Carrier = {};
+  ensureHubOnCarrier(carrier, parent);
+  return getHubFromCarrier(carrier);
+}
+
+/** TODO docs */
```

Comment:
l: shall we add them now, or wait till after?

Reply:
Good point, will add a little bit of docs!